hack/tools/Makefile

```diff
@@ -121,3 +121,7 @@ $(CONVERSION_VERIFIER): $(BIN_DIR) go.mod go.sum ## Build a local copy of conver
 SETUP_ENVTEST := $(BIN_DIR)/setup-envtest
 $(SETUP_ENVTEST): $(BIN_DIR) go.mod go.sum ## Build a local copy of setup-envtest.
 	go build -tags=capibmtools -o $@ sigs.k8s.io/controller-runtime/tools/setup-envtest
+
+GOVULNCHECK := $(BIN_DIR)/govulncheck
+$(GOVULNCHECK): ## Install govulncheck.
+	GOBIN=$(abspath $(BIN_DIR)) go install golang.org/x/vuln/cmd/govulncheck@v1.0.1
```

Comment:
why this is not following the other tools pattern? and also tag the version in the go.mod file

Reply:
@mkumatag The build approach would need to get the package beforehand before building it but then running go mod tidy would clean the mod files as it is not used explicitly in code.

Reply:
for that you need to make an entry here in this file - https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/hack/tools/tools.go

Reply:
fixed.